sites/docs/src/content/docs/tutorials/tests_and_test_data/test_data.md

```diff
@@ -58,6 +58,14 @@ In contrast, if your module requires many tests, steps, or heavy CPU usage, you
 
 11. The test data pull request requires a review to be merged.
 
+12. Once you've had your pull request merged in into `nf-core/test-datasets`, you are good to go and you can refer to the newly uploaded file(s) for your test(s). Use `params.modules_testdata_base_path` to point to the test data directory in your module, and append to it the path to the file(s) you need. Here is an example using the `fasta` file from `sarscov2`:
```

Comment:
```suggestion
12. Once you've had your pull request merged in into `nf-core/test-datasets`, you are good to go and you can refer to the newly uploaded file(s) for your test(s). Use `params.modules_testdata_base_path` to point to the test data directory in your module, and append to it the path to the file(s) you need. Here is an example using the `fasta` file from `sarscov2`:
```

I don't get waht you mean by 'to point to the test data directory in your module'

It points to the remote URL (on GitHbub) or local source ( a clone of teh repo) of all the  testr data

Can you rephrase?